s3watcher/lib/Upload.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
          const headers = {
             'Content-Length': s3Event.size,
             'Content-Type': 'application/octet-stream',
-            'X-Gu-Media-Key': config.apiKey
+            'X-Gu-Media-Key': config.stage == "PROD" ? "xxx" : config.apiKey
```

Comment:
Brain fart trying to hide the key in logs, this will do that but also not work at all so :crying_cat_face: